src/store/CategoricalDisplayStore.js

```diff
@@ -180,9 +180,12 @@ export default class CategoricalDisplayStore {
             return filteredItems;
         }
 
-        const reg = RegExp(this.filterText.trim(), 'i');
+
+        // regex to check all separate words by comma, should exist in the string
+        const reg = RegExp(`.*(${this.filterText.split(' ').filter(x => x !== '').map(text => text.trim()).join(')+.*(')})+`, 'i');
```

Comment:
@brucebinary  by considering the binary-static code I change the regex to treat same as binary-static. thats the thing I change in the code to solve the issue

Reply:
in this regex I search in the text to be sure it contain all the `input search text` which user type